README.md

```diff
@@ -57,11 +136,5 @@ TODO:
 - creating and using template
```

Comment:
URLs bitte mit [text](URL) formatieren! Wenn wir in Markdown schreiben, sollten wir auch Markdowns verwenden.

Reply:
Also:
\[creating and using template\]\(https://expressjs.com/en/guide/using-template-engines.html\)

Reply:
Changed links to correct markdown format.